flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml

```diff
@@ -179,6 +179,7 @@
             <UseTemplate Name="ASOBO_LIGHTING_Potentiometer_Emissive_Template">
```

Comment:
We could consider making a custom template in the future which takes the bus straight up instead of needing to write RPN

Reply:
@Benjozork Like a separate bus-template or a template where one could specify a bus?

Reply:
Every emmisive in any Airbus we simulate will be powered off one of the elec system busses, so it should be part of the emmisive templates. Good refactor to do on a rainy day.